packages/patient-portal/src/features/dashboard/Procedures/ProceduresSection.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Stack, Typography } from '@mui/material';
+import { HeartPulse } from 'lucide-react';
+import { AccordionSection } from '../../../components/AccordionSection';
+import { StyledCircularProgress } from '../../../components/StyledCircularProgress';
+import { useProceduresQuery } from '@api/queries/useProceduresQuery';
+import { ProcedureCard } from './ProcedureCard';
+
+export const ProceduresSection = () => {
+  const { data: procedures, isLoading } = useProceduresQuery();
+
+  const plannedProcedures = procedures?.filter(procedure => !procedure.completed) || [];
+  const completedProcedures = procedures?.filter(procedure => procedure.completed) || [];
```

Comment:
cool trick for this with lodash but i just like it because it iterates once for big lists so not that valid here
```suggestion
const [plannedProcedures, unplannedProcedures] = partition(procedures, ({completed}) => completed)
```

Reply:
nice - that's much nicer to read than the reduce method. I think I'll leave as is here since it's a small list and save a dependency.